addons/l10n_vn_edi/models/sinvoice.py

```diff
@@ -0,0 +1,69 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, fields
+
+
+# Invoice template that needs to be passed to Sinvoice and will determine the format of the resulting
+# invoice pdf on their system
+class SInvoiceTemplate(models.Model):
+    _name = 'sinvoice.template'
+    _description = 'SInvoice template'
+
+    name = fields.Char(
+        string="Template Code",
+        required=True,
+    )
+    template_invoice_type = fields.Selection(
+        selection=[
+            # Circular 32
+            ('01GTKT', '01GTKT - Value-added invoice'),
+            ('02GTTT', '02GTTT - Sales invoice'),
+            ('07KPTQ', '07KPTQ - Sales invoice for organizations and individuals in the non-tariff zone'),
+            ('03XKNB', '03XKNB - Warehouse release note cum internal transportation'),
+            ('04HGDL', '04HGDL - Warehouse release note sent to sales agent'),
```

Comment:
This Circular 32 is deprecated, accountant in Vietnam use the 78 one instead
![image](https://github.com/odoo/odoo/assets/56789189/02e76af4-dd10-4894-bcff-16a8a77cae9b)
Sorry for the vietnamese image, i can't find an english one, hehe

Reply:
I see, so I can safely remove these choices from the list?
This is something the Functional Consultant in charge told me too (that we may not need them), but as I wasn't sure it was no longer used I kept it just in case. Thanks!

Reply:
Of course, the api by sinvoice only work with the circular 78.
For the latest sinvoice development api, download from here , every 2-3 month they updated with some changes, i'm not sure whether they have english documentation or not
![image](https://github.com/odoo/odoo/assets/56789189/0d17dbd5-ee64-40dc-9ccc-de2159d72739)

Reply:
[tailieu_mo_ta_webservice_hoadondientu_doitac_v2.35_public.docx](https://github.com/odoo/odoo/files/14927130/tailieu_mo_ta_webservice_hoadondientu_doitac_v2.35_public.docx)

Reply:
They don't have an English version indeed.
Thank you, I will remove the additional/unnecessary choices with the next push

Reply:
Yeah if you need support about api i can help . I worked with sinvoice for over a year, of course just some advices and some experience because i can't give all of my company product about sinvoice in this PR hehe